DB/classification/models.py

```diff
@@ -57,6 +52,32 @@ def __init__(self, model_type="default", created_at=datetime.datetime.utcnow(),
         self.model = model
         self.id = _id
 
+    @classmethod
+    def reload_db_info(cls):
+        """
+        :summary: Reloads which database/user pair to use (for testing). Defaults to seads/seadapi when no file is present (old behavior)
+        """
+        db_info = None
+        try:
+            db_info = open("db_info", 'r')
```

Comment:
Please write unit tests mocking or stubbing the database. Integration tests are good too, but shouldn't replace unit tests and should be runable in our canonical environment.

My plan is to run our normal deployment scripts on circleci. After that, we should run integration tests which should work without configuration.